serverless/compiler-lambda/index.js

```diff
@@ -298,23 +300,22 @@ function asyncExecSolc(inputStringified, solcPath) {
   });
 }
 
-exports.handler = async (event) => {
+exports.handler = awslambda.streamifyResponse(async (event, responseStream, _context) => {
+  let output;
   try {
-    return {
-      success: true,
-      body: await useCompiler(
-        event.version,
-        event.solcJsonInput,
-        event.forceEmscripten
-      ),
-    };
+    output = await useCompiler(
+      event.version,
+      event.solcJsonInput,
+      event.forceEmscripten
+    );
   } catch (e) {
-    return {
-      success: true,
-      body: e.message,
-    };
+    output = { error: e.message };
   }
-};
+  console.debug("Compilation output: ", output);
+
+  const outputBlob = new Blob([JSON.stringify(output)]);
```

Comment:
Continuing this comment: https://github.com/ethereum/sourcify/pull/1340#discussion_r1560488122

like this:
```
if (weight(outputBlob) > 20MB ) {
  output = { error: "specific error for this case" }
  return or throw
}
```

Reply:
Ah nice idea! That should work out! Thanks for the hint